src/frontend/src/common/components/library/data_subview/components/CreateNSTreeModal/components/RadioLabel/strings.ts

```diff
@@ -39,3 +39,23 @@ export const pathogenStrings: PathogenStringsType<RadioLabelStrings> = {
       "Good for identifying samples most closely related to the selected samples among all samples in GISAID and your CZ GEN EPI samples.",
   },
 };
+
+/* TODO: (ehoops)
+ * This object contains copy updates that are behind the
+ * tree_location_filter feature flag.  When the feature flag is removed,
+ * these should be moved into the pathogenStrings object above.
+ */
+export const tempLocationFilterCopyUpdates: Partial<RadioLabelStrings> = {
+  nonContextualizedDescription:
+    "Builds tree with only samples of interest (defined by location, date, and lineage).",
+  nonContextualizedGoodFor:
+    "Good for seeing viral diversity in the public repository that may not be captured by your own sampling effort.",
+  nonContextualizedNotRecommended:
+    "Not recommended for epidemiological interpretation due to lack of visibility into viral diversity outside of the selections below and omission of closely-related samples.",
+  overviewBestFor:
+    "Best for generating a summary tree of samples of interest, in the context of genetically similar GISAID samples.",
+  overviewGoodFor2:
+    "Good for specifying sample location, collection date, or lineages instead of the defaults used for your CZ GEN EPI weekly build.",
+  targetedDescription:
+    "Builds tree with selected samples (from the sample table or the ID box below) and closely-related samples, at a ratio of roughly 1:2.",
+};
```

Comment:
I'm not sure how we want to go about updating copy while using the pathogen-specific strings.  In an awkward twist, I'm not sure all of this will end up being pathogen-specific.  Still, I think it would be good to think about whether this update method makes sense or if there's a more clear way to do it.

Reply:
Yeah, I agree. This seems like it could get awkward as pathogen-specific stuff gets more complicated. But I think this is a perfectly reasonable way to deal with the issue, especially until our pathogen-specific stuff starts to gel. That said, if we start to have interaction between different multiple, in-development feature flags and a pathogen flag at once, wooooof, that could get real hairy. But we can cross that bridge when we come to it. For now this is clear and gets the job done, so I like it.